### ISSUE TITLE
Broken when used in upper_third or middle_center

### ISSUE THREAD

**Issue by ivanovd** [raywo/MMM-NowPlayingOnSpotify#33]
When showing the component in the upper_third or middle_center it shows the cover art in full screen and nothing else.
![maxthonsnap20181123000132](https://user-images.githubusercontent.com/4889583/48922957-f3ff8a00-eeb2-11e8-8294-3452c2a05a1a.png)

**Comment by ejay-ibm** [comment]
Yes I have the same issue.

**Comment by abssorb** [comment]
Also happening with fullscreen_above

**Comment by clancyjr** [comment]
agreed. it goes bananas in certain positions. solution: leave it right or left. blah.

**Comment by guikubivan** [comment]
> Also happening with fullscreen_above
> 
> A workaround for me is to use .NPOS_albumcover set to 50%

Great solution. To be newbie friendly and exact, I added to `css/custom.css`:
```
.NPOS_albumCover {
 width: 50%;
}
```
